examples/with-react-intl/pages/_document.tsx

```diff
@@ -0,0 +1,46 @@
+import Document, {
```

Comment:
I'm not sure if I'm doing this correctly w/ next.js. I'm trying to hydrate a static config-like databag from server to client (in this case, `locale`). Doing it in `_app` doesn't seem to work for client-side navigation since `req` is `undefined` after the initial load